packages/core/src/view/mixins/CellsMixin.ts

```diff
@@ -2714,13 +2714,16 @@ export const CellsMixin: PartialType = {
   /**
    * Returns true if the given cell is moveable. This returns {@link cellsMovable}
    * for all given cells if {@link isCellLocked} does not return true for the given
```

Comment:
**suggestion:** let's take this opportunity to improve the entire JSDoc of the method 
```suggestion
   * Returns `true` if the given cell is movable. This returns {@link cellsMovable}
   * for all given cells if {@link isCellLocked} does not return `true` for the given
```